scripts/index.js

```diff
@@ -0,0 +1,182 @@
+const mainSkills = document.getElementById("main-skills");
+const mainPortfolio = document.getElementById("main-portfolio");
+const mainVideo = document.getElementById("main-video");
+const mainPrice = document.getElementById("main-price");
+const mainFeedback = document.getElementById("main-feedback");
+const headerItemsRightPartList = document.getElementById("header-items-right-part-list");
+headerItemsRightPartList.addEventListener('click', (event) => {
+  if (event.target.closest('.header__items-right-part-list-element-link-skills')) {
+    mainSkills.scrollIntoView({ behavior: 'smooth' });
+  }
+  else if (event.target.closest('.header__items-right-part-list-element-link-portfolio')) {
+    mainPortfolio.scrollIntoView({ behavior: 'smooth' });
+  }
+  else if (event.target.closest('.header__items-right-part-list-element-link-video')) {
+    mainVideo.scrollIntoView({ behavior: 'smooth' });
+  }
+  else if (event.target.closest('.header__items-right-part-list-element-link-price')) {
+    mainPrice.scrollIntoView({ behavior: 'smooth' });
+  }
+  else if (event.target.closest('.header__items-right-part-list-element-link-contacts')) {
+    mainFeedback.scrollIntoView({ behavior: 'smooth' });
+  };
+
+});
+
+
+const headerItemsRightPartMenuButton = document.getElementById("header-items-right-part-menu-button");
+const asideButton = document.getElementById("aside-button");
+const asideWrapper = document.getElementById("aside-wrapper");
+headerItemsRightPartMenuButton.addEventListener('click', () => {
+  asideWrapper.style.visibility = 'visible';
+  asideWrapper.style.opacity = 1;
+});
+document.addEventListener('click', (event) => {
+  if (event.target.classList.contains('aside-wrapper') || event.target.closest('.aside__button')) {
+    asideWrapper.style.visibility = 'hidden';
+    asideWrapper.style.opacity = 0;
+  }
+});
+asideWrapper.addEventListener('click', (event) => {
```

Comment:
We can do this logic event without JS. We can add links instead of buttons and make some refs for them in order to move into target place on the page.